### PR TITLE
Adding ECS ES monitoring

### PR DESCRIPTION
**Pull request by minniux**
Adding ES monitoring for ECS clusters, using the elasticsearch exporter and SG to allow access to the elasticsearch cluster to be monitored

ES monitoring is optional, adding an additional container for the exporter in the monitoring task

Related issue: https://github.com/skyscrapers/engineering/issues/127

### CODE REVIEW

File: monitoring/main.tf [hunk 1]
```diff
@@ -212,6 +261,46 @@ resource "aws_security_group_rule" "allow_ecs_node_monitor_out" {
...
+  count             = "${var.enable_es_exporter ? 1 : 0 }"
+  type              = "egress"
+  from_port         = "${var.es_exporter_port}"
+  to_port           = "${var.es_exporter_port}"
+  protocol          = "tcp"
+  security_group_id = "${var.ecs_sg}"
+  self              = true
+}
+
+resource "aws_security_group_rule" "allow_es_exportor_external" {
```
Comment (iuriaranda) [inline monitoring/main.tf]: Resource name has a typo: `exportor` -> `exporter`

File: monitoring/main.tf [hunk 2]
```diff
@@ -212,6 +261,46 @@ resource "aws_security_group_rule" "allow_ecs_node_monitor_out" {
...
+  count                    = "${var.enable_es_exporter ? 1 : 0 }"
+  type                     = "ingress"
+  from_port                = "${var.es_exporter_port}"
+  to_port                  = "${var.es_exporter_port}"
+  protocol                 = "tcp"
+  security_group_id        = "${var.ecs_sg}"
+  source_security_group_id = "${var.es_sg}"
+}
+
+resource "aws_security_group_rule" "allow_es_exprtor_external_out" {
```
Comment (iuriaranda) [inline monitoring/main.tf]: Same here: `exprtor`

File: monitoring/variables.tf [hunk 3]
```diff
@@ -101,6 +101,62 @@ variable "cloudwatch_exporter_memory_reservation" {
   default = "100"
 }
 
+variable "enable_es_exporter" {
+  default = "false"
```
Comment (iuriaranda) [inline monitoring/variables.tf]: I think this should be unquoted to make it an explicit bool

**Review by iuriaranda — APPROVED** [review]
Just a few typos, for the rest it looks ok to me

File: monitoring/task-definitions/monitoring.json [hunk 4]
```diff
@@ -1,4 +1,3 @@
-[
       {
         "name":"s3_download",
         "image": "mesosphere/aws-cli:latest",
```
Comment (duboisph) [inline monitoring/task-definitions/monitoring.json]: maybe replace this by our own image? `skyscrapers/awscli`

**Review by duboisph — APPROVED** [review]
Lgtm, apart from the typos already mentioned

**Pull request merged**